docs/user_guides/single_table/custom_constraints.rst

```diff
@@ -0,0 +1,263 @@
+.. _custom_constraints:
+
+Defining Custom Constraints
+===========================
+
+In some cases, the predefined constraints do not cover all your needs. 
+In such scenarios, you can use ``CustomConstraint`` to define your own 
+logic on how you would like to apply it to your data. There are three 
+main functions that you can create:
+
+- ``transform`` which is responsible for the forward pass when using ``transform`` strategy.
+- ``reverse_transform`` which defines how to reverse the transformation of the ``transform``.
+- ``is_valid`` which indicates which rows satisfy the constraint and which ones do not.
+
+Let's look at a demo dataset:
+
+.. ipython:: python
+    :okwarning:
+
+    from sdv.demo import load_tabular_demo
+
+    employees = load_tabular_demo()
+    employees
+
+The dataset defined in :ref:`_single_table_constraints` contains basic details about employees.
+We will use this dataset to demonstrate how you can create your own constraint. 
+
+
+Using the ``CustomConstraint``
+------------------------------
+
+Let's consider the following example where we wish to generate synthetic data and we want
+a particular column, ``salary`` for example, to be a multiple of a *base* value, e.g. 500. 
+In other words, the ``salary`` increments by 500. We will define ``transform`` and 
+``reverse_transform`` methods to make sure our data satisfy our constraints.
+
+We can achieve our goal by performing transformations in a 2 step process:
+
+- Divide ``salary`` by the base unit (500). This transformation makes it easier to model the data
```

Comment:
wording sounds strange: "...transformation makes the job easier for the model, since..." ?